flags/src/main/java/com/yahoo/vespa/flags/JacksonArraySerializer.java

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.flags;
+
+import java.util.List;
+
+/**
+ * @author freva
+ */
+public class JacksonArraySerializer<T> implements FlagSerializer<List<T>> {
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public List<T> deserialize(RawFlag rawFlag) {
+        return (List<T>) JsonNodeRawFlag.fromJsonNode(rawFlag.asJsonNode()).toJacksonClass(List.class);
+    }
+
+    @Override
+    public RawFlag serialize(List<T> value) {
+        return null;
```

Comment:
This must be implemented

I see we're missing tests of RawFlag serialization/deserialization that should be added... It would have been great to have tests of: Flag + JSON string gets deserialized to expected values, and serialized back to the same JSON.

Could you add ListFlag to InMemoryFlagSource?

Reply:
Oops! Implemented.

Added basic serialization/deserialization to `FlagsTest`

Added `ListFlag` to `InMemoryFlagSource`